src/matroid.lean

```diff
@@ -8,36 +8,18 @@ variables {α : Type*} {β : Type*} [decidable_eq α]
 namespace finset
 /- For mathlib? -/
 lemma inter_of_subset {A B : finset α} (h : A ⊆ B) : A ∩ B = A :=
-lattice.inf_of_le_left h
-
-lemma subset_iff_sdiff_eq_empty {x y : finset α} : x ⊆ y ↔ x \ y = ∅ :=
```

Comment:
I started with deleting a bunch of these just to see how many may have made it into mathlib since this was written, but had to add a few back again (maybe not enough yet even).